src/pages/create-or-edit-lesson/CreateOrEditLesson.tsx

```diff
@@ -162,8 +162,11 @@ const CreateOrEditLesson = () => {
   })
 
   useEffect(() => {
-    if (id) void fetchDataLesson(id)
-  }, [id, fetchDataLesson])
+    if (id) {
+      void fetchDataLesson(id)
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@Tolik170 are we allowed to suppress ts warnings?

Reply:
@TSlashDreamy In some cases, for example, if I want to function inside useEffect executes only one time at first render, the dependency array should be empty, here I guess it want fetchDataLesson to be part of this array